### PR TITLE
Prettify README

### PR DESCRIPTION
**Pull request by Cheesebaron**

### CODE REVIEW

**Comment by Cheesebaron** [comment]
To reviewers, please check spelling, content etc. A lot of the links etc. I removed could go on the webpage instead.

File: README.md [hunk 1]
```diff
@@ -1,161 +1,38 @@
...
-* [Xamarin forums](http://forums.xamarin.com)
-* [Slack](https://xamarinchat.herokuapp.com/) join the #mvvmcross channel after you are in
-
-### Documentation and Examples
-
-**The [MvvmCross-Samples](https://github.com/MvvmCross/MvvmCross-Samples) repo contains the latest samples. See the [MvvmCross Wiki](https://github.com/MvvmCross/MvvmCross/wiki) for additional articles and information.**
+👀 Check out [mvvmcross.com](https://www.mvvmcross.com) to get started with MvvmCross 👀
 
-**Samples and Videos:**
+MvvmCross is a cross-platform MVVM framework. It enables developers to create apps using the MVVM pattern on Xamarin and Windows (UWP & WPF). This allows for better code sharing by allowing you to share behavior and business logic between platforms.
```
Comment (nmilcoff) [inline README.md]: Can we list here the same platforms the docs enumerates?
Xamarin.iOS, Xamarin.Android, Xamarin.Mac, Xamarin.Forms, Universal Windows Platform (UWP) and Windows Presentation Framework (WPF)

File: README.md [hunk 2]
```diff
@@ -1,161 +1,38 @@
...
-* [Evolve 2013 talk](http://xamarin.com/evolve/2013#session-dnoeeoarfj)
-* [NDC 2013 talk](http://slodge.blogspot.co.uk/2013/06/ndc2013-mobile-mvvm-and-xamarin-talks.html)
-* [Xaminar - 2012](http://slodge.blogspot.com/2012/12/mvvmcross-video-presentation-xaminar.html)
-* [MvvmCross and Azure - 2012](http://slodge.blogspot.co.uk/2012/11/azure-to-wpmonodroidmonotouch-video-of.html)
+Among the features MvvmCross provides are:
 
+- ViewModel to View bindings using own customizable binding engine, which allows you to create own binding definitions for own custom views
+- ViewModel to ViewModel navigation, helps you share behavior on how and when to navigate
+- Inversion of Control through Dependency Injection and Property Injection
+- Plugin framework, which lets you plug-in cool stuff like GPS Location, Localization, Sensors, Binding Extensions and a huge selection of 3rd party community plug-ins
```
Comment (nmilcoff) [inline README.md]: This page lists some more features: https://www.mvvmcross.com/documentation/getting-started/getting-started

The missing ones are:
- MVVM architecture pattern
- Platform specifics support (Android Support packages, iOS side menu)
- Unit test helpers
- Complete flexibility - your app is King!

File: README.md [hunk 3]
```diff
@@ -1,161 +1,38 @@
...
 
-**The Tip Calc tutorial:**
```
Comment (nmilcoff) [inline README.md]: Doesn't it worth to keep a small reference to this tutorial? It's definitely a good first step to start coding

File: README.md (hunk 2 above)
Comment (Cheesebaron) [inline README.md]: MVVM architecture is already implied by the previous text.

Complete flexibility is already mentioned in the paragraph just be low the list.

We could add Platform specifics. I don't think Unit Test helpers is the real selling point here though.

File: README.md (hunk 3 above)
Comment (Cheesebaron) [inline README.md]: People should be able to find it through the docs. Which is why, the docs are mentioned twice in the README.

File: README.md (hunk 1 above)
Comment (Cheesebaron) [inline README.md]: Sure. It covers the same, little bit more verbose though.

File: README.md (hunk 2 above)
Comment (nmilcoff) [inline README.md]: Yes, you're right. But in my opinion the more bullets we list, the more we sell

Comment (Cheesebaron) [inline README.md]: What I read is:

> The more crap we list, the more we sell

Like a real sales person! 🥇

Comment (nmilcoff) [inline README.md]: 🤑

File: README.md [hunk 4]
```diff
@@ -226,3 +103,105 @@ Become a sponsor and get your logo on our README on Github with a link to your s
...
+
+## Contributing code
+
+We are happy to receive Pull Requests adding new features and solving bugs. As for new features, please contact us before doing major work. To ensure you are not working on something that will be rejected due to not fitting into the roadmap or ideal of the framework.
+
+### Git setup
+
+Since Windows and UNIX-based systems differ in terms of line endings, it is a very good idea to configure git autocrlf settings.
+
+On *Windows* we recommend setting `autocrlf` to `true`.
```
Comment (kjeremy) [inline README.md]: `core.autocrlf`

File: README.md [hunk 5]
```diff
@@ -226,3 +103,105 @@ Become a sponsor and get your logo on our README on Github with a link to your s
...
+
+```
+git config --global core.autocrlf true
+```
+
+On *Mac* we recommend setting `autocrlf` to `input`.
```
Comment (kjeremy) [inline README.md]: `core.autocrlf`

**Pull request merged**